.github/workflows/_build.yaml

```diff
@@ -90,6 +90,27 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ github.token }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@8c0edbc76e98fa90f69d9a2c020dcb50019dc325   # v2.2.1
```

Comment:
We try not to use third-party Actions. Can we use `docker` directly instead?

Reply:
I will switch to use docker CLI inside `run` instead.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/b3b86c7d2e1307ff3a76b10615886555ecd207a3